.travis/linux..install.sh

```diff
@@ -15,3 +15,9 @@ else
 fi
 
 sudo apt-get install -y $PACKAGES
+
+# Fix header
```

Comment:
@jasp00 should this be a cmake task so that it doesn't show up for non-travis builds too?